bootstrap.sh

```
@@ -242,6 +241,195 @@ function getUbuntuVersion {
+    fi
+}
+
+function stepInstallNeoVim() {
+    if ! [ -e "$(command -v nvim)" ]; then
+        logStep "Installing NeoVim..."
+        # Adds repo for latest neovim version
+        runCmdAndLog add-apt-repository -y ppa:neovim-ppa/stable
+        runCmdAndLog ${APT_CMD} update
+        runCmdAndLog ${APT_INSTALL} neovim
+        # Set neovim as default vim
+        runCmdAndLog update-alternatives --set vi $(which nvim)
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)